### ISSUE TITLE
When hovering on a speaker card it enlarges and makes a shadow

### ISSUE THREAD

**Issue by brittanyjoiner15** [brittanyjoiner15/eui-event-template#37]
When hovering on a card for a speaker, the card is enlarged and there is a shadow on the element.

![](https://user-images.githubusercontent.com/16166290/192384526-50363674-9397-457f-9192-aed9d27a68c9.png)

Alternatively, you could pick something from here: https://blog.avada.io/css/card-hover-effects

Basically just need some kind of animation that would make the card become more obvious when people are hovering over it, but nothing too dramatic. 

If you take this issue, please comment with some ideas of what you'd like to implement here, and maybe a demo or example before implementing! (I recognize the description might not be as clear as it could be.)

**Comment by brandonmcconnell** [comment]
@brittanyjoiner15 I'd like to take this one!

My idea is to scale it up to make the card larger, offset the rotation by something small like 15deg, and add a shadow or glow to the card. 🙂

Does this page support light and dark mode?

**Comment by binnyle** [comment]
Hi @brittanyjoiner15 , I love your website, I was thinking we could try something like this

https://codepen.io/ovdojoey/pen/BomKyZ

Maybe with a lighter variation since the site primary color is dark, it will create an amazing contrast and made the card stand out. 

Thanks!

**Comment by abitsalihu** [comment]
Hey, I would happily help with this issue and implement the design you guys are looking for. :)

**Comment by brittanyjoiner15** [comment]
These are all great ideas! @brandonmcconnell i've assigned you to the FAQ tab issue, and i appreciate the enthusiasm but we only assign folks to one issue at a time unless there's an exception where they're very closely connected issues. So I'll have to ask that you finish that other one and once your PR is merged, you'll be welcome to pick up another issue!

In the meantime @BinniesLite you haven't been assigned anything yet so I am happy to assign this to you! I like that hover effect! Subtle but still noticeable, and the contrast is a good idea. 

@abitsalihu I'm assigning this to Binnies since they've commented first, but feel free to pair with them on this if you have some ideas!

**Comment by abitsalihu** [comment]
@brittanyjoiner15 That's fine. @BinniesLite If you need any help, feel free to msg me. :)

**Comment by brittanyjoiner15** [comment]
@abitsalihu thanks for understanding! Feel free to grab another issue if you'd like or if you have some ideas, you can propose an issue and I'll review it and assign it to you!

**Comment by binnyle** [comment]
Thanks, I'll get this done in the Hacktoberfest @brittanyjoiner15 and absolutely @abitsalihu, I'll message you if I run into any trouble.

**Comment by brittanyjoiner15** [comment]
@BinniesLite awesome!! if you can just keep me posted on progress every couple of days that would be great! Totally fine to wait until Oct 1st, but don't want this to sit stale for too long if we can help it!

**Comment by binnyle** [comment]
Definitely, I'll keep you update on the progress @brittanyjoiner15

**Comment by brittanyjoiner15** [comment]
@BinniesLite thank you so much!

**Comment by akshitsangwan** [comment]
Hey @brittanyjoiner15, just wanted to thank you for sharing this super amazing website with us and the ideas were too great! I will try to implement and add some of em in my portfolio as well:)

**Comment by binnyle** [comment]
Hi @brittanyjoiner15, would you prefer a lighter tone like this or keep it dark?

![image](https://user-images.githubusercontent.com/89041460/192985947-206fe447-a7bd-472b-80fe-531ad504e32b.png)

**Comment by brittanyjoiner15** [comment]
@akshitsangwan thanks for saying that, definitely please do!!

**Comment by brittanyjoiner15** [comment]
@BinniesLite i like the lighter tone!! gives good contrast!

**Comment by binnyle** [comment]
Awesome, I'll made a PR in October 1st

**Comment by brittanyjoiner15** [comment]
@BinniesLite sounds great! I'll be ready to review and merge!
